pkgs/applications/video/gpu-screen-recorder/gpu-screen-recorder-gtk.nix

```diff
@@ -50,23 +51,30 @@ stdenv.mkDerivation {
     wayland
   ];
 
-  preFixup = let
-    gpu-screen-recorder-wrapped = gpu-screen-recorder.override {
-      inherit wrapperDir;
-    };
-  in ''
-    gappsWrapperArgs+=(--prefix PATH : ${wrapperDir})
-    gappsWrapperArgs+=(--suffix PATH : ${lib.makeBinPath [ gpu-screen-recorder-wrapped ]})
-    # we also append /run/opengl-driver/lib as it otherwise fails to find libcuda.
-    gappsWrapperArgs+=(--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libglvnd ]}:/run/opengl-driver/lib)
-  '';
+  preFixup =
+    let
+      gpu-screen-recorder-wrapped = gpu-screen-recorder.override {
+        inherit wrapperDir;
+      };
+    in
+    ''
+      gappsWrapperArgs+=(--prefix PATH : ${wrapperDir})
+      gappsWrapperArgs+=(--suffix PATH : ${lib.makeBinPath [ gpu-screen-recorder-wrapped ]})
+      gappsWrapperArgs+=(--prefix LD_LIBRARY_PATH : ${
```

Comment:
We cannot use patchelf?

Reply:
I'm not exactly sure how that would be implemented..